model/gcp/registry.yaml

```diff
@@ -195,3 +195,42 @@ groups:
         brief: >
           The hostname of a GCE instance. This is the full value of the default or [custom hostname](https://cloud.google.com/compute/docs/instances/custom-hostname-vm).
         examples: ['my-host1234.example.com', 'sample-vm.us-west1-b.c.my-project.internal']
+
+  - id: registry.gcp.gen_ai
+    type: attribute_group
+    display_name: Generative AI Attributes
+    brief: >
+      This documents defines attributes for Google generative AI systems.
+    stability: development
+    attributes:
+      - id: gcp.gen_ai.operation.config
+        stability: development
+        type: string
+        brief: >
+          Operation-specific request configuration options for Google generative AI operations.
+
+          The string value should be a valid JSON dictionary. The JSON dictionary should include
+          only an allowlisted subset of the request configuration that does not overlap with
+          configuration that has already been captured elsewhere through standard Semantic Conventions.
```

Comment:
would you mind moving it to the note (to keep the brief short)?

Reply:
Done.